examples/activities/user_joined_room_activity.py

```diff
@@ -0,0 +1,36 @@
+import asyncio
+import logging.config
+from pathlib import Path
+
+from symphony.bdk.core.activity.user_joined_room import UserJoinedRoomActivity, UserJoinedRoomContext
+from symphony.bdk.core.config.loader import BdkConfigLoader
+from symphony.bdk.core.service.message.message_service import MessageService
+from symphony.bdk.core.symphony_bdk import SymphonyBdk
+
+async def run():
+    config = BdkConfigLoader.load_from_symphony_dir("/Users/reed.feldman/devx/test-bot-python/resources/config.yaml")
```

Comment:
usually the examples use the symphony dir (i.e ~/.symphony) so we don't have to change the path to configuration

Reply:
Let me revise the path to the default.